shared/version/api.go

```diff
@@ -115,6 +115,7 @@ var APIExtensions = []string{
 	"container_protection_delete",
 	"unix_priv_drop",
 	"pprof_http",
+	"proxy_protocol",
```

Comment:
Hm, can't we name this `proxy_proxy_protocol` or `proxy_haproxy_protocol` or `proxy_haproxy`. I like to keep the generic api extension prefix to be meaningful, like `storage_<something>_<something>`.